src/mmw/js/src/draw/templates/window.html

```diff
@@ -1,2 +1,99 @@
 <h1>Select Area</h1>
-<p>Draw tools go here</p>
+<p id="draw-pane-header-description">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Duis ac pharetra nibh. Nullam gravida massa non egestas
+    pellentesque.
+</p>
+<div id="draw-pane-buttons">
```

Comment:
Since these are all pretty similar, consider passing the view the `DrawTools` and generating these based on them:

```javascript
{% for drawTool in drawTools %}
    <div class="btn btn-lg btn-block draw-tool-button" type="button"
         data-toggle="dropdown" aria-expanded="true"
         id={{ id based on drawTool }} >
        <p class="draw-button-title pull-left">
            {{ drawTool }}
        </p>
        {% if activeDrawTool != drawTool %}
            <span class="draw-button-icon caret pull-right"></span>
        {% endif %}
        <br />
        <p class="draw-button-subtitle pull-left">
            Lorem ipsum dolor sit amet, consectetur
        </p>
    </div>
    {% if activeDrawTool == drawTool %}
      <div class="btn btn-lg btn-block draw-tool-button-slideout" type="button"
            data-toggle="dropdown" aria-expanded="true">
             <i class="fa fa-times pull-right reset-draw-button"></i>
             <div id={{region id based on drawTool }}></div>
             </div>
     {% endif %}
{% endfor %}
```